Project.toml

```
@@ -1,7 +1,7 @@
 name = "MixedModels"
 uuid = "ff71e718-51f3-5ec2-a782-8ffcbfa3c316"
 author = ["Phillip Alday <me@phillipalday.com>", "Douglas Bates <dmbates@gmail.com>"]
-version = "4.30.1"
+version = "4.30.2"
```

```suggestion
version = "4.31.0"
```

this is technically a new feature, so under SemVer rules, it gets a minor version bump. Patch bumps are for bug fixes.

(Also this is why there are often no NEWS entries for patch releases -- often they're addressing tiny corner case bugs, so we don't feel the need to explain much there. That said, if we ever discuss a critical, inaccurate results bug, then I would include that in NEWS.)